shared/util_linux.go

```diff
@@ -36,6 +36,7 @@ import (
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <sys/un.h>
+#include <acl/libacl.h>
```

Comment:
Yes. FWIW both the sample code I found on the internet and get/setfacl include both acl.h and libacl.h. What's the issue with including libacl.h?